test/functional/feature_block.py

```diff
@@ -160,7 +160,7 @@ def run_test(self):
             blockname = f"for_invalid.{TxTemplate.__name__}"
             self.next_block(blockname)
             badtx = template.get_tx()
-            if TxTemplate != invalid_txs.InputMissing:
+            if TxTemplate != invalid_txs.InputMissing and TxTemplate != invalid_txs.SizeExactly64:
```

Comment:
In 8ee48e40aa63e698bd414bd93340e8d34355a6d3 _consensus: make 64 bytes transactions invalid_: I'm confused by this test change. Was the test incorrect before?

I managed to apply this change to 7ccdcc1215ab27fb2739646261817daffeabd7a2 _[test] Separate 64B and 63B tx size tests_ instead and it still passes. So that's a better place, though a comment would be useful.

Reply:
It is necessary because `sign_tx()` will populate the scriptSig and the transaction won't be 64-byte anymore. I don't think 7ccdcc1 would be a better place for this change, since the failure is due to enforcing the check by consensus which is only done in the commit where the change is made.